examples/md-flexible/src/TimeDiscretization.h

```diff
@@ -28,10 +28,13 @@ namespace TimeDiscretization {
  * @param particlePropertiesLibrary The particle properties library for the particles in the container.
  * @param deltaT The time step width.
  * @param globalForce Base force value to which every particle is reset.
+ * @param fastParticlesThrow When true throws an exception if particles moved too far for verlet technique
+ * (>skin/2/rebuildFrequency).
```

Comment:
```suggestion
 * @param fastParticlesThrow When true throws an exception if particles moved too far to ensure the static containers' skins remain valid (>skin/2/rebuildFrequency). 
 * Does nothing if dynamic containers are enabled.
```
Probably is not correct for clang format